source/server/admin/utils.cc

```diff
@@ -56,6 +56,27 @@ bool filterParam(Http::Utility::QueryParams params, Buffer::Instance& response,
   return true;
 }
 
+// Helper method to get the histogram_buckets parameter. Returns false if histogram_buckets query
+// param is found and value is not "cumulative" or "disjoint", true otherwise.
+bool histogramBucketsParam(const Http::Utility::QueryParams& params, Buffer::Instance& response,
+                           HistogramBucketsMode& histogram_buckets_mode) {
```

Comment:
nit: consider returning absl::Status which can encode success/failure and an optional error message here, rather than modifying a `response` param.

See https://github.com/envoyproxy/envoy/blob/35f8d59b1172a85f02dd119791a9ef42c728cfa8/source/server/admin/logs_handler.h#L41 for a pattern for a similar scenario.

Reply:
I changed the boolean return value to absl::Status and removed the `response` param